package.json

```diff
@@ -121,7 +121,8 @@
     "temp-write": "^5.0.0",
     "tsd": "^0.17.0",
     "typescript": "~4.4",
-    "verdaccio": "^5.1.1"
+    "verdaccio": "^5.1.1",
+    "vue-template-compiler": "workspace:*"
```

Comment:
I don't really understand why we need this 🤔

Reply:
I put the explanation there: https://github.com/transloadit/uppy/pull/3802/files#diff-4bea83be50434c7f99646e7178ae936953bfe18cd00c64066fd67442c8b7aab7R5
If we don't have a `vue-template-compiler` package at the root level, the `vue-template-compiler` of the Vue2 example will be installed to the root `node_modules` dir instead of the `examples/vue/node_modules` dir, and because we have Vue 3 installed in the root `node_modules` dir, it picks it up instead of the Vue 2 installed in the  `examples/vue/node_modules` dir. I think that's the fault of this package to not list Vue 2 in its (peer) dependencies: https://github.com/vuejs/vue/blob/dev/packages/vue-template-compiler/package.json